tests/unit/BusinessTypeConfirm.spec.ts

```diff
@@ -13,16 +13,16 @@ describe('Business Type Confirm component', () => {
   it('works as expected when there initial business type confirm is false', async () => {
     wrapper = wrapperFactory(
       BusinessTypeConfirm,
-      { 
+      {
```

Comment:
When I ran `npm run lint`, these issues were fixed automatically.

To all devs: please lint locally before committing code.